resources/views/cruds/datagrids/filters/datagrid-filter.blade.php

```diff
@@ -60,7 +60,7 @@
                                     @include('cruds.datagrids.filters._tag')
                                 @elseif ($field['type'] === 'select')
                                     @include('cruds.datagrids.filters._select')
-                                @elseif ($field['label'] === 'Location')
+                                @elseif ($field['label'] === 'Location' || $field['label'] === 'Family' || $field['label'] === 'Race' || $field['label'] === 'Organisation')
                                     @include('cruds.datagrids.filters._location')
```

Comment:
I feel like the `_location` file should be renamed, since it's not limited to just locations. I also feel  like we don't _need_ another file, but instead it could simply be an option from the way the Filter class is built. For example, in the `CharacterFilter` class, add an option `'withChildren' => true`, and in the `_select` file, simply check if the option is enabled ;-) that way we don't need to add a new if in this file each time we add the option to a filter.